docs/src/rules/default-param-last.md

```diff
@@ -41,6 +41,34 @@ Examples of **correct** code for this rule:
 /* eslint default-param-last: ["error"] */
 
 function f(a, b = 0) {}
+
+function g(a, b = 0, c = 0) {}
+```
+
+:::
+
+This rule additionally supports TypeScript type syntax.
+
+Examples of **incorrect** TypeScript code for this rule:
+
+::: incorrect
+
+```ts
+/* eslint default-param-last: ["error"] */
+
+function h(a = 0, b: number) {}
```

Comment:
Looks like the error location isn't underlined?

![image](https://github.com/user-attachments/assets/efdabe49-5285-4f48-9cdb-47dcd0fc99c4)

Reply:
And perhaps we should remove the "Open in Playground" button because ts examples don't work in ESLint Playground?

Reply:
👍 filed https://github.com/eslint/eslint.org/issues/709 and added a check for `ts` language to skip adding the button. I'll add a root-level comment for the error location issue.